docs/reference/components/sinks/azure_blob.cue

```diff
@@ -0,0 +1,201 @@
+package metadata
+
+components: sinks: azure_blob: {
+	title: "Azure Blob Storage"
+
+	classes: {
+		commonly_used: true
+		delivery:      "at_least_once"
+		development:   "beta"
+		egress_method: "batch"
+		service_providers: ["Azure"]
+		stateful: false
+	}
+
+	features: {
+		buffer: enabled:      true
+		healthcheck: enabled: true
+		send: {
+			batch: {
+				enabled:      true
+				common:       true
+				max_bytes:    10485760
+				timeout_secs: 300
+			}
+			compression: {
+				enabled: true
+				default: "gzip"
+				algorithms: ["none", "gzip"]
+				levels: ["none", "fast", "default", "best", 0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+			}
+			encoding: {
+				enabled: true
+				codec: {
+					enabled: true
+					default: null
+					enum: ["ndjson", "text"]
+				}
+			}
+			request: {
+				enabled:                    true
+				concurrency:                50
+				rate_limit_duration_secs:   1
+				rate_limit_num:             250
+				retry_initial_backoff_secs: 1
+				retry_max_duration_secs:    10
```

Comment:
I think the default here is actually 3600.

Reply:
@jszwedko with the value of 3600 of retry_max_duration_secs `make check-docs` command fails with error 

```
./scripts/check-docs.sh
Skipping cue files format validation - reserved for CI
Validating cue files correctness...
Failed!

components.sinks.azure_blob.features.send.request.retry_max_duration_secs: invalid value 3600 (out of bound <=255):
    ./docs/reference/components/sinks/azure_blob.cue:45:33
```

I also searched for other places in docs for `retry_max_duration_secs` and everywhere value is 10

Reply:
Aha, that's not right, it should be a `uint64`. The default is set here:

https://github.com/timberio/vector/blob/629e3b434fa6a57fbb21b69afd605fd558055f20/src/sinks/util/service.rs#L200-L204

I'll update the component cue file. It is likely that other sinks don't have the correct default either.

Reply:
I opened https://github.com/timberio/vector/pull/8033 to fix this. We can merge that first and then fix the value here.

Reply:
Sure, thanks for this explanation

Reply:
I merged #8033 . Would you want to merge master into this branch update the docs to drop the values here that are defaulted? I attempted to do that myself but it looks like I don't have permission.

Reply:
@jszwedko Merged and updated